### ISSUE TITLE
misplacement of plot,coastline-method axis labels

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1736]
An image for our argoFloats paper looks as below.  The LH panel is from `oce::plot,coastline-method` and the right one is from `argoFloats::plot`; notice the offset in the labels in the former (see especially the equator and prime meridian.

"There's a pale sky in the east... all the stars are in the west" -- Nanci Griffith

**Comment by dankelley** [comment]
Hm, I'm stupid, using `format`:
```R
> X<-c(-180,-135,-90,-45,0,45,90,135,180)
> format(X)
[1] "-180" "-135" " -90" " -45" "   0" "  45" "  90" " 135" " 180"
```

**Comment by dankelley** [comment]
Fixed in "develop" commit 7d322dea0d38912a45a92097fa02efba6c551fad

He says, "There's barbed wire at all these exits/And there ain't no place in Belfast for that kid to go" -- Nanci Griffith
